lib/Retcon/Network/Server.hs

```diff
@@ -102,13 +106,17 @@ spawnServer cfg n = do
         sock   <- socket ctx Rep
         bind sock zmq_conn
         db     <- initBackend (PGOpts pg_conn)
-        return $  ServerState ctx sock cfg db
+        -- Setup ekg
+        ekgStore  <- initialiseMeters cfg
+        ekgServer <- Ekg.forkServerWith ekgStore "localhost" 8888
+        return $  ServerState ctx sock cfg db ekgServer
 
     stop :: ServerState -> IO ()
     stop state = do
         closeBackend $ state ^. serverStore
         close        $ state ^. serverSocket
         term         $ state ^. serverContext
+        killThread   $ Ekg.serverThreadId $ state ^. serverEkgServer
```

Comment:
is there a pre-defined way in the ekg package to stop a server?

if you just `killThread` (assuming from control.concurrent), an exception will be raised in the ekg server thread, which is created by:

```
forkFinally (startServer store host port) $ \ r ->
        case r of
            Left e  -> throwTo me e
....
```

I think that exception will be re-raised in this retcon server thread. Not sure if that's what we want.